.circleci/config.yml

```diff
@@ -19,69 +30,86 @@ commands:
           command: |
             ./opt/readies/bin/getpy3
             BREW_NO_UPDATE=1 ./opt/system-setup.py
-            # git clone git://github.com/antirez/redis.git --branch 6.0.1; (cd redis; make malloc=libc -j $(nproc); make install); redis-server --version
+            # ./opt/readies/bin/getredis -v 6 --force
```

Comment:
make sure Redis is built with `malloc=libc`.

Reply:
Hi @rafie, looks great. I'm pretty sure `libc` is the default on macOS